book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,146 @@
+| [abscissa_core](https://github.com/iqlusioninc/abscissa/tree/develop/) | 0.5.2 
+| [gumdrop](https://github.com/murarth/gumdrop) | 0.7.0 
+
+
+## Out of scope
+
+| Name | Version | Reason | Notes
+|------| ------- | -----  | -----
+| [proptest](https://github.com/altsysrq/proptest) | [0.10.1](https://github.com/altsysrq/proptest/releases/tag/v0.10.1) | Testing
+| proptest-derive | 0.3.0 | Testing
+| [tracing](https://github.com/tokio-rs/tracing) | 0.1.36 | Tracing
+| [tracing-futures](https://github.com/tokio-rs/tracing) | 0.2.5 | Tracing 
+| [lazy_static](https://github.com/rust-lang-nursery/lazy-static.rs) | [1.4.0](https://github.com/rust-lang-nursery/lazy-static.rs/releases/tag/1.4.0) 
+| [static_assertions](https://github.com/nvzqz/static-assertions-rs) | [1.1.0](https://github.com/nvzqz/static-assertions-rs/releases/tag/v1.1.0) 
+| [thiserror](https://github.com/dtolnay/thiserror) | [1.0.37](https://github.com/dtolnay/thiserror/releases/tag/1.0.37) | Error handling
+| [dirs](https://github.com/soc/dirs-rs) | 4.0.0 |
+| displaydoc | 0.2.3 | Docs 
+| [metrics](https://github.com/metrics-rs/metrics) | 0.20.1 | Metrics
+| [once_cell](https://github.com/matklad/once_cell) | [1.15.0](https://github.com/matklad/once_cell/releases/tag/v1.15.0) 
+| [regex](https://github.com/rust-lang/regex) | [1.6.0](https://github.com/rust-lang/regex/releases/tag/1.6.0) 
+| [tracing-error](https://github.com/tokio-rs/tracing) | 0.2.0 | Tracing
+| [num_cpus](https://github.com/seanmonstar/num_cpus) | [1.13.1](https://github.com/seanmonstar/num_cpus/releases/tag/v1.13.1) | Trivial use 
+| [rlimit](https://github.com/Nugine/rlimit/) | [0.8.3](https://github.com/Nugine/rlimit//releases/tag/v0.8.3) 
+| [tempfile](https://github.com/Stebalien/tempfile) | [3.3.0](https://github.com/Stebalien/tempfile/releases/tag/v3.3.0) 
+| [color-eyre](https://github.com/yaahc/color-eyre) | [0.6.2](https://github.com/yaahc/color-eyre/releases/tag/v0.6.2) | Error handling
+| [tracing-subscriber](https://github.com/tokio-rs/tracing) | 0.3.15 | Logging
+| [log](https://github.com/rust-lang/log) | [0.4.17](https://github.com/rust-lang/log/releases/tag/0.4.17) 
+| [metrics-exporter-prometheus](https://github.com/metrics-rs/metrics) | 0.11.0 | Metrics
+| [sentry-tracing](https://github.com/getsentry/sentry-rust) | [0.27.0](https://github.com/getsentry/sentry-rust/releases/tag/0.27.0) | Tracing
+| [toml](https://github.com/alexcrichton/toml-rs) | [0.5.9](https://github.com/alexcrichton/toml-rs/releases/tag/0.5.9) 
+| [tracing-appender](https://github.com/tokio-rs/tracing) | 0.2.2 | Tracing
+| [tracing-journald](https://github.com/tokio-rs/tracing) | 0.3.0 | Tracing
+| [atty](https://github.com/softprops/atty) | [0.2.14](https://github.com/softprops/atty/releases/tag/0.2.14) 
+| [rand_chacha](https://github.com/rust-random/rand) | [0.3.1](https://github.com/rust-random/rand/releases/tag/0.3.1) | Testing
```

Comment:
@conradoplg is this a consensus-critical cryptographic library, or is it only used in testing?